src/test/kotlin/uk/gov/justice/digital/hmpps/supportadditionalneedsapi/integration/messaging/CuriousALNTriggerEventTest.kt

```diff
@@ -188,12 +115,60 @@ class CuriousALNTriggerEventTest : IntegrationTestBase() {
     // Given
     val prisonNumber = randomValidPrisonNumber()
     stubGetTokenFromHmppsAuth()
-    stubGetCurious2LearnerAssessments(prisonNumber, createTestALNAssessment(prisonNumber))
     prisonerInEducation(prisonNumber)
     anElSPExists(prisonNumber)
 
     // When
     val curiousReference = UUID.randomUUID()
+    createALNAssessmentMessage(prisonNumber, curiousReference, hasNeed = true)
+
+    val reviewScheduleEntity = reviewScheduleRepository.findFirstByPrisonNumberOrderByUpdatedAtDesc(prisonNumber)
+    Assertions.assertThat(reviewScheduleEntity!!.deadlineDate).isNull()
+    Assertions.assertThat(reviewScheduleEntity.status).isEqualTo(ReviewScheduleStatus.SCHEDULED)
+  }
+
+  // The person has a challenge and an aln and a plan creation schedule then has an aln need removed via a message
+  // test that the person still has a need and that the plan creation schedule remains SCHEDULED
+  @Test
```

Comment:
Added two new tests to check that the ALN need being removed DOESN'T change the schedules providing the person already had a need.